client/state/ui/editor/reducer.js

```diff
@@ -63,6 +64,15 @@ export function isLoading( state = false, action ) {
 	return state;
 }
 
+export function isIframeLoaded( state = false, action ) {
+	switch ( action.type ) {
+		case EDITOR_IFRAME_LOADED:
+			return get( action, 'isIframeLoaded', false );
```

Comment:
The `setEditorIframeLoaded` action creator ensured that `action.isIframeLoaded` is always present and is a boolean. I prefer to use `_.get` or `?.` only in cases where a field is really optional.

Reply:
> The `setEditorIframeLoaded` action creator ensured that `action.isIframeLoaded` is always present and is a boolean

There is the chance that the action may not have been invoked by the time the reducer is called ( `setEditorIframeLoaded` is only called when the editor is presented, but we need to guarantee that the reducer always returns a value to Electron's even if the user hasn't interacted with the editor yet) . So I think this makes sense, as-is?

Otherwise I could be missing something, if there's a more idiomatic way to write this. (Was following the pattern used for the existing `isLoading` reducer in this file.)

Reply:
If the reducer is called with any other action, it will just return the `state` parameter unchanged. Acting as identity function.

When the Redux store is being initialized, all reducers are called with `reducer( undefined, { type: '@@redux/INIT' } )`. Then the undefined `state` param has a chance to default to `false`.

Specifying a default value for the `state` parameter is the idiomatic way to initialize a state value.